internal/internal_workflow_testsuite.go

```diff
@@ -1622,7 +1622,31 @@ func (env *testWorkflowEnvironmentImpl) SideEffect(f func() ([]byte, error), cal
 	callback(f())
 }
 
-func (env *testWorkflowEnvironmentImpl) GetVersion(changeID string, minSupported, maxSupported Version) Version {
+func (env *testWorkflowEnvironmentImpl) GetVersion(changeID string, minSupported, maxSupported Version) (retVerion Version) {
+	if _, ok := env.expectedMockCalls[mockMethodForGetVersion]; ok {
+		// GetVersion is mocked
+		args := []interface{}{changeID, minSupported, maxSupported}
+		// below call will panic if mock is not properly setup.
+		mockRet := env.mock.MethodCalled(mockMethodForGetVersion, args...)
+		m := &mockWrapper{name: mockMethodForGetVersion, fn: mockFnGetVersion}
+		if mockFn := m.getMockFn(mockRet); mockFn != nil {
+			executor := &activityExecutor{name: mockMethodForGetVersion, fn: mockFn}
+			reflectValues := executor.executeWithActualArgsWithoutParseResult(nil, args)
+			if len(reflectValues) != 1 || !reflect.TypeOf(reflectValues[0].Interface()).AssignableTo(reflect.TypeOf(retVerion)) {
+				panic(fmt.Sprintf("mock of GetVersion has incorrect return type, expected workflow.Version, but actual is %T (%v)",
+					reflectValues[0].Interface(), reflectValues[0].Interface()))
+			}
+			return reflectValues[0].Interface().(Version)
+		} else {
+			if len(mockRet) != 1 || !reflect.TypeOf(mockRet[0]).AssignableTo(reflect.TypeOf(retVerion)) {
```

Comment:
Could this be `reflect.TypeOf((*Version)(nil))` instead?

Also, fix spelling of `retVerion`.

Reply:
remove retVersion and use DefaultVersion instead.